components/items/ItemsGrid/ItemsGridImage.vue

```diff
@@ -117,9 +117,6 @@ const nftStack = computed(() =>
 const isAvailbleToBuy = computed(() =>
   nftStack.value.some((nft) => Number(nft.price) > 0),
 )
-const anyAvailableForListing = computed(() =>
-  nftStack.value.some((nft) => !Number(nft.price)),
-)
```

Comment:
why remove it's still used in this component

Reply:
If you check the latest nuxt branch, you will found that it's not used anymore.